buildchain/buildchain/salt_tree.py

```diff
@@ -85,8 +87,7 @@ def task(self) -> types.TaskDict:
         task = self.basic_task
         task.update({
             'title': self._show,
-            'doc': 'Generate the nginx config to serve a common static '
-                   'container registry.',
+            'doc': self._TASK_DOC,
```

Comment:
Since the `_TASK_DOC` is really similar to the docstring of the class, we may want to use `self.__doc__` instead of a dedicated variable.